src/main/java/homer/model/temperaturechangers/AbstractTemperatureChanger.java

```diff
@@ -87,4 +108,25 @@ public final void setState(final DeviceState state) {
         }
     }
 
+    @Override
+    public final void plug(final Outlet outlet) {
+        this.power.setOutlet(outlet);
+    }
+
+    @Override
+    public final PoweredDeviceInfo getPowerInfo() {
+        return new PoweredDeviceInfoImpl(this.power.getMaxConsumption(), this.power.getOutlet());
+    }
+
+    /**
+     * 
+     * @param deltaTime the delta time elapsed from last tick.
+     */
+    protected void updateConsumption(final Duration deltaTime) {
+        final double maxConsumption = this.getPowerInfo().getMaxConsumption();
+        final double hours = DurationConverter.toHours(deltaTime);
+        final double intensity = Math.sin(hours * 0.1); 
```

Comment:
Guess this will be changed, in this way you'll get VERY small values (You're calculating the sine of 1/10 of the hours passed, if deltaTime is 24H you'll get a value of 0.04, which multiplied for the hours gives 1, so you're actually increasing the consumption of 1W every hour which is definetly too slow. 
Also you should bind the power absorption with the intensity of the Device, also you could use a loglinear function.

Try by calculating it in this way:
```
final double maxConsumption = this.getPowerInfo().getMaxConsumption();
final double hours = DurationConverter.toHours(deltaTime);
final double logIntensity = Math.log10(this.getIntensity() + 1.0); //1.0 can be adjusted, the higher it is, the faster consumption will grow. It can't be less than 1.0 otherwise you'll get negative values for the consumption.
final double newConsumption = instantConsumption + maxConsumption * logIntensity * hours;
this.instantConsumption = Limit.clamp(newConsumption, this.getPowerInfo().getMinConsumption, maxConsumption);
```

With this function you should get pretty good values in general, and it should grow along with intensity. I assumed the intensity to be between 0.0 and 1.0 tho. If you intend to use another range of values you'll need to scale intensity to be between 0.0 and 1.0 to be effective.

In this way the function is loglinear (nlogn).
You can also add fluctuations by subtracting small values (e.g. 0.01 + Math.random() * 0.05) to this.getIntensity().